src/deluge/gui/menu_item/runtime_feature/settings.cpp

```diff
@@ -57,10 +59,18 @@ Submenu subMenuAutomation{
 };
 
 std::array<MenuItem*, RuntimeFeatureSettingType::MaxElement - kNonTopLevelSettings> subMenuEntries{
-    &menuDrumRandomizer,       &menuMasterCompressorFx, &menuFineTempo,           &menuQuantize,
-    &menuPatchCableResolution, &menuCatchNotes,         &menuDeleteUnusedKitRows, &menuAltGoldenKnobDelayParams,
-    &subMenuAutomation,        &menuDevSysexAllowed,
-};
+    &menuDrumRandomizer,
+    &menuMasterCompressorFx,
+    &menuFineTempo,
+    &menuQuantize,
+    &menuPatchCableResolution,
+    &menuCatchNotes,
+    &menuDeleteUnusedKitRows,
+    &menuAltGoldenKnobDelayParams,
+    &subMenuAutomation,
+    &menuDevSysexAllowed,
+    &menuHighlightIncomingNotes,
+    &menuDisplayNornsLayout};
```

Comment:
trailing comma should probably be here so we get the dangling brace

Reply:
Fixed :)